run_in_compat_layer_env.sh

```diff
@@ -29,8 +29,8 @@ fi
 if [ ! -z ${EESSI_SOFTWARE_SUBDIR_OVERRIDE} ]; then
     INPUT="export EESSI_SOFTWARE_SUBDIR_OVERRIDE=${EESSI_SOFTWARE_SUBDIR_OVERRIDE}; ${INPUT}"
 fi
-if [ ! -z ${EESSI_ACCELERATOR_TARGET} ]; then
-    INPUT="export EESSI_ACCELERATOR_TARGET=${EESSI_ACCELERATOR_TARGET}; ${INPUT}"
+if [ ! -z ${EESSI_ACCELERATOR_TARGET_OVERRIDE} ]; then
```

Comment:
@trz42 This is why I was asking about where these environment variables get set, this should be using the override mechanism